timeseries/src/main/scala/com/criteo/cuttle/timeseries/TimeSeriesApp.scala

```diff
@@ -662,31 +671,38 @@ private[timeseries] case class TimeSeriesApp(project: CuttleProject, executor: E
         )
       }
 
-      if (request.headers.get(h"Accept").contains(h"text/event-stream"))
-        sse(IO { watchState() }, (s: WatchedState) => IO(getFocusView(s)))
-      else
-        watchState() match {
-          case Some(watchedState) => Ok(getFocusView(watchedState))
-          case None               => BadRequest
+      request
+        .readAs[Json]
+        .flatMap { json =>
+          json
+            .as[CalendarFocusQuery]
+            .fold(
+              df => IO.pure(BadRequest(s"Error: Cannot parse request body: $df")),
+              query => {
+                val jobs = Try(query.jobs.filter(_.nonEmpty)).toOption
```

Comment:
I don't get why we need a Try here

Reply:
Me neither, just took that from previous source.
That pattern is also used in couple places of this files, removed it on my edits though.
Also there is a global `allIds` that I used instead of recomputing `project.jobs.all.map(_.id)` each time